### PR TITLE
WIP Mutated copy from Vault integration test

### PR DESCRIPTION
**Pull request by benjaminbollen**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/386)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by benjaminbollen** [comment]
prints too many iterations of "enter command"; needs addressing

**Comment by benjaminbollen** [comment]
also does not work

**Comment by benjaminbollen** [comment]
addresses #385

File: tests/lib.rs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                Err(why) => panic!("couldn't spawn maidsafe_vault: {}", why.description()),
+                Ok(process) => process,
+            });
+    thread::sleep_ms(1000);
+
+    for i in 1..num_of_nodes {
+        println!("---------- starting node {} --------------", i);
+        processes.push(match Command::new("./target/debug/examples/simple_key_value_store")
+            .arg("-n").stdout(Stdio::piped()).spawn() {
+                    Err(why) => panic!("couldn't spawn maidsafe_vault: {}", why.description()),
```
Comment (dirvine) [inline tests/lib.rs]: This library should not use the term maidsafe and probably not vault either IMHO, it leads to the slippery slope of coupling.

File: tests/lib.rs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+use std::error::Error;
+use std::io::Read;
+
+#[test]
+// This test requires the executable simple_key_value_store is present at the same place of the test get executed
+// also it depends on the printout. if such printout is changed / muted, this test needs to be updated
+fn executable_test() {
+    let mut processes = Vec::new();
+    let num_of_nodes = 8;
+    // the first vault must be run in zero_membrane mode
```
Comment (dirvine) [inline tests/lib.rs]: I think this `first` notion will give a lot more edge cases than just this actually.

File: tests/lib.rs (hunk 1 above)
Comment (benjaminbollen) [inline tests/lib.rs]: sorry, yes this is a remnant from copying this test from maidsafe_vaults; I will update the comments

**Comment by Viv-Rajkumar** [comment]
Closing this PR for now. Please raise it again if required.
